scripts/wsmux.sh

```diff
@@ -36,14 +39,81 @@ wsmux() {
 	while (( $# > 0 )); do
 		case $1 in
 			-h|--help)
-				echo "Usage: wsmux [OPTION]... [CATKIN_WORKSPACE]"
+				echo "Usage: wsmux [OPTION]... [WORKSPACE]"
 				echo "Quick catkin workspace switcher."
 				echo ""
 				echo "Option		GNU long option		Meaning"
+				echo "-b [WORKSPACE]	--bind			Bind an SSH client to a workspace"
+				echo "-c		--connect		Select the SSH client's workspace"
 				echo "-h		--help			Display the help menu"
-				echo "-l		--list			List the detected catkin workspaces"
-				echo "-s		--show			Show the selected catkin workspace"
+				echo "-l		--list			List the detected workspaces"
+				echo "-s		--show			Show the selected workspace"
+				SELECTION="false"
+				shift 1
+				;;
+			-b|--bind)
+
+				# Ensure that a client has connected via SSH
+				if [[ -z "$SSH_CLIENT" ]]; then
```

Comment:
Perhaps there should be an optional  "default" or "local" line in the wsmux.conf for when not over SSH. So in our case it will source mil_ws if non on ssh or if ip address is not on config